shared/wallets/send-form/participants/to-field.js

```diff
@@ -235,18 +234,16 @@ const styles = Styles.styleSheetCreate({
 
   // ToStellarPublicKey
   toStellarPublicKey: {
-    height: 52,
```

Comment:
does this work in mobile

Reply:
No, the send form has a bunch of problems so I didn't work on it since we have another ticket this sprint

Reply:
I don't think we can work on these things In isolation. You'll just break the other side

Reply:
I don't think this fix will affect that ticket, bigger layout changes are needed to get it working on mobile